.github/workflows/JOSS-pdf.yml

```diff
@@ -0,0 +1,26 @@
+on:
+  push:
+    paths:
+      - docs/JOSS/**
+
+jobs:
+  paper:
+    runs-on: ubuntu-latest
+    name: JOSS Draft
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Build draft JOSS PDF
+        uses: openjournals/openjournals-draft-action@master
+        with:
+          journal: joss
+          # This should be the path to the paper within your repo.
+          paper-path: paper.md
```

Comment:
not that it matters that much anymore, but shouldn't this be `docs/JOSS/paper.md` ?

Reply:
Yup, thanks